modules/face/src/getlandmarks.cpp

```diff
@@ -0,0 +1,252 @@
+Copyright (C) 2009-2015, NVIDIA Corporation, all rights reserved.
+Copyright (C) 2010-2013, Advanced Micro Devices, Inc., all rights reserved.
+Copyright (C) 2015, OpenCV Foundation, all rights reserved.
+Copyright (C) 2015, Itseez Inc., all rights reserved.
+Third party copyrights are property of their respective owners.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+  * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+  * Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation
+    and/or other materials provided with the distribution.
+
+  * Neither the names of the copyright holders nor the names of the contributors
+    may be used to endorse or promote products derived from this software
+    without specific prior written permission.
+
+This software is provided by the copyright holders and contributors "as is" and
+any express or implied warranties, including, but not limited to, the implied
+warranties of merchantability and fitness for a particular purpose are disclaimed.
+In no event shall copyright holders or contributors be liable for any direct,
+indirect, incidental, special, exemplary, or consequential damages
+(including, but not limited to, procurement of substitute goods or services;
+loss of use, data, or profits; or business interruption) however caused
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.
+*/
+
+#include "opencv2/face.hpp"
+
+#include <ctime>
+
+using namespace std;
+namespace cv{
+namespace face{
+void FacemarkKazemiImpl :: readSplit(ifstream& is, splitr &vec)
+{
+    is.read((char*)&vec, sizeof(splitr));
+}
+void FacemarkKazemiImpl :: readLeaf(ifstream& is, vector<Point2f> &leaf)
+{
+    unsigned long size;
+    is.read((char*)&size, sizeof(size));
+    leaf.resize(size);
+    is.read((char*)&leaf[0], leaf.size() * sizeof(Point2f));
+}
+void FacemarkKazemiImpl :: readPixels(ifstream& is,int index)
+{
+    is.read((char*)&loaded_pixel_coordinates[index][0], loaded_pixel_coordinates[index].size() * sizeof(Point2f));
+}
+
+bool FacemarkKazemiImpl :: load(string filename){
+    if(filename.empty()){
+        String error_message = "No filename found.Aborting....";
+        CV_Error(Error::StsBadArg, error_message);
+        return false;
+    }
+    ifstream f(filename.c_str(),ios::binary);
+    if(!f.is_open()){
+        String error_message = "No file with given name found.Aborting....";
+        CV_Error(Error::StsBadArg, error_message);
+        return false;
+    }
+    cout<<"Loading....."<<endl;
```

Comment:
Avoid cout in sourcecode... thats not needed at all. most users want functions to work silently.